spec/helpers/federal_register_documents_helper_spec.rb

```diff
@@ -29,6 +29,25 @@
     it { is_expected.to eq result }
   end
 
+  describe 'federal register links' do
```

Comment:
Because this is testing the `link_to_federal_register_advanced_search` method, the test description & subject should include that method:
```ruby
describe `#link_to_federal_register_advanced_search` do
  subject(:link_to_federal_register_advanced_search) do
...etc
```